### PR TITLE
Add global drops

### PR DESCRIPTION
**Pull request by aapomm**
### Summary

Add project records as available drops in LiquidEnabledResource

### Other Information

If there's anything else that's important and relevant to your pull
request, mention that information here. This could include
benchmarks, or other information.

Thanks for contributing to Dradis!

### Copyright assignment

Collaboration is difficult with commercial closed source but we want
to keep as much of the OSS ethos as possible available to users
who want to fix it themselves.

In order to unambiguously own and sell Dradis Framework commercial
products, we must have the copyright associated with the entire
codebase. Any code you create which is merged must be owned by us.
That's not us trying to be a jerks, that's just the way it works.

Please review the [CONTRIBUTING.md](https://github.com/dradis/dradis-ce/blob/master/CONTRIBUTING.md)
file for the details.

You can delete this section, but the following sentence needs to
remain in the PR's description:

> I assign all rights, including copyright, to any future Dradis
> work by myself to Security Roots.

### Check List

- [ ] Added a CHANGELOG entry

### CODE REVIEW

File: app/controllers/concerns/liquid_enabled_resource.rb [hunk 1]
```diff
@@ -33,6 +33,10 @@ def project_assigns
     authorize! :use, project
 
     result['project'] = ProjectDrop.new(project)
+    result['issues'] = project.issues.map { |issue| IssueDrop.new(issue) }
+    result['nodes'] = project.nodes.user_nodes.map { |node| NodeDrop.new(node) }
+    result['evidences'] = project.evidence.map { |evidence| EvidenceDrop.new(evidence) }
+    result['tags'] = project.tags.map { |tags| TagDrop.new(tags) }
```
Comment (caitmich) [inline app/controllers/concerns/liquid_enabled_resource.rb:39]:
```suggestion
    result['tags'] = project.tags.map { |tag| TagDrop.new(tag) }
```

File: spec/support/revision_shared_examples.rb [hunk 2]
```diff
@@ -61,6 +61,11 @@
 
       expect do
         visit project_trash_path(current_project)
+
+        within '#trash' do
```
Comment (caitmich) [inline spec/support/revision_shared_examples.rb:65]: unrelated?

Comment (aapomm) [reply spec/support/revision_shared_examples.rb:65]: Sort of? But this change is needed because this example expects the instance to only have 1 of the item in the trash. If for some reason there are other items in the trash, the spec fails. In the case of the liquid shared examples, it creates multiple test records that break this example.

**Comment by caitmich** [comment]
Not implemented
